demo/video_gpuaccel_demo.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+
+import cv2
+import mmcv
+import ffmpegcv
+import torch
+import tqdm
+import numpy as np
+from torchvision.transforms import functional as F
+from mmdet.apis import init_detector
+from mmdet.datasets.pipelines import Compose
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='MMDetection video demo')
+    parser.add_argument('video', help='Video file')
+    parser.add_argument('config', help='Config file')
+    parser.add_argument('checkpoint', help='Checkpoint file')
+    parser.add_argument(
+        '--device', default='cuda:0', help='Device used for inference')
+    parser.add_argument(
+        '--score-thr', type=float, default=0.3, help='Bbox score threshold')
+    parser.add_argument('--out', type=str, help='Output video file')
+    parser.add_argument('--show', action='store_true', help='Show video')
+    parser.add_argument('--nvdecode', action='store_true', help='Use NVIDIA decorder')
+    parser.add_argument(
+        '--wait-time',
+        type=float,
+        default=1,
+        help='The interval of show (s), 0 is block')
+    args = parser.parse_args()
+    return args
+
+
+def prepare_cfg(model, ori_wh):
+    cfg = model.cfg
+    w, h = ori_wh
+    cfg.data.test.pipeline[0].type = 'LoadImageFromWebcam'
+    test_pipeline = Compose(cfg.data.test.pipeline)
+    data = {'img':np.zeros((h, w, 3), dtype=np.uint8)}
+    data = test_pipeline(data)
+    data_template = data['img_metas'][0].data
+    mean = torch.tensor(data_template['img_norm_cfg']['mean'])
+    std = torch.tensor(data_template['img_norm_cfg']['std'])
+    return data_template, mean, std
+
+
+def prepare_img(data_template, mean, std, frame):
+    assert frame.shape==data_template['pad_shape']
+    frame_cuda = torch.from_numpy(frame.astype(np.float32)).cuda()
```

Comment:
Why no need to resize. doesn't seem to run `test_pipeline`?

Reply:
I've done the resize  and padding in ffmpegcv